addons/hr_attendance/static/src/less/hr_attendance.less

```diff
@@ -9,15 +9,15 @@
 
 .oe_hr_attendance_status {
     font-size: 1.2em;
-    margin-right: .3em;
+    margin-right: .1em;
     display: inline-block;
     color: @odoo-brand-secondary;
 
     &.oe_hr_attendance_status_green {
-        color: @brand-success;
+        color: @brand-success !important;
```

Comment:
I'll make "oe_hr_attendance_status_green" an id instead of a class since it has more weight for the css attributes.

id matching has weight of 100
10 weight per class matched
1 per tag matched

Highest weight has highest priority when it comes to attributes.